src/components/sbb-calendar/sbb-calendar.tsx

```diff
@@ -64,50 +57,50 @@ interface CalendarKeyboardNavigationParameters {
   verticalOffset: number;
 }
 
-@Component({
-  shadow: true,
-  styleUrl: 'sbb-calendar.scss',
-  tag: 'sbb-calendar',
-})
-export class SbbCalendar implements ComponentInterface {
+export const events = {
+  dateSelected: 'date-selected',
+};
+
+@customElement('sbb-calendar')
+export class SbbCalendar extends LitElement {
+  public static override styles: CSSResult = Style;
+
   /** If set to true, two months are displayed */
-  @Prop() public wide = false;
+  @property({ type: Boolean }) public wide = false;
 
   /** The minimum valid date. Takes Date Object, ISOString, and Unix Timestamp (number of seconds since Jan 1, 1970). */
-  @Prop() public min: Date | string | number;
+  @property() public min: Date | string | number;
```

Comment:
If the consumer uses the attribute, you'll always receive a string here when the user sets the attribute. Are you fine with it? (same thing for the other properties with the same type)

Reply:
I tested it with all options and they always get parsed right, so I'd say we're fine